addons/website_event_exhibitor/views/event_type_views.xml

```diff
@@ -5,10 +5,11 @@
         <field name="model">event.type</field>
         <field name="inherit_id" ref="website_event.event_type_view_form"/>
         <field name="arch" type="xml">
-            <xpath expr="//div[@name='website_menu']" position='after'>
-                <div class="row mt16" attrs="{'invisible': [('website_menu', '=', False)]}">
-                    <label class="col" for="exhibitor_menu"/> <field name="exhibitor_menu"/>
-                </div>
+            <xpath expr="//field[@name='website_track_proposal' or @name='website_menu'][last()]" position='after'>
+                <span attrs="{'invisible': [('website_menu', '=', False)]}">
+                    <label for="exhibitor_menu" string="Exhibitors Menu Item"/>
+                    <field name="exhibitor_menu"/>
+                </span>
```

Comment:
By renaming the exhibitor_menu label, we have different labels in event type and in event for the same concept. 
'Showcase Exhibitors' vs 'Exhibitors Menu Item'
Maybe align both ? check with @LNAodoo